js_modules/dagster-ui/packages/ui-core/src/assets/usePaginatedAssetEvents.tsx

```diff
@@ -70,42 +64,38 @@ export function usePaginatedAssetEvents(
           cursor,
           before,
           after: afterParam,
-          eventTypeSelector: params.status ?? MaterializationHistoryEventTypeSelector.ALL,
+          eventTypeSelectors: params.statuses ?? [
+            AssetEventHistoryEventTypeSelector.MATERIALIZATION,
+            AssetEventHistoryEventTypeSelector.OBSERVATION,
+            AssetEventHistoryEventTypeSelector.FAILED_TO_MATERIALIZE,
+          ],
         },
       });
       setLoading(false);
 
       const asset = data?.assetOrError.__typename === 'Asset' ? data?.assetOrError : null;
 
-      const {materializations, observations} = clipEventsToSharedMinimumTime(
```

Comment:
It looks like this PR doesn't remove `clipEventsToSharedMinimumTime` but I bet it's unused now and could be deleted! Good riddance :-)